src/side-panel/side-panel.js

```diff
@@ -0,0 +1,187 @@
+import React, { useCallback } from 'react'
+import { Box, Flex } from 'theme-ui'
+import { Button, Row, Column } from '@carbonplan/components'
+import { ArrowThin } from '@carbonplan/icons'
+
+import SidePanelAttachment from './side-panel-attachment'
+const SidePanel = ({
+  children,
+  tooltip,
+  onClose,
+  footer,
```

Comment:
did you consider an alternative where one or more `SidePanelFooter`s could simply be the final children of the `SidePanel`, rather than as a prop? that feels a little more composable, but maybe there are good reasons not to do it that way, or it's too complicated to get the layout right? thinking something like...

```
<SidePanel>
  <Stuff...>
  <SidePanelFooter/>
</SidePanel>
```

Reply:
Documenting conversation over Zoom: because the scrolling content must be wrapped in an element with `flex: '1 1 auto'` , the footer content must be distinct from other content in `children` and  a `footer` prop is a natural way to handle this.